bower.json

```diff
@@ -17,6 +17,7 @@
     "o-grid": "^4.2.3",
     "o-icons": ">=4.4.2 <6",
     "o-labels": "^4.2.1",
-    "o-spacing": "^2.0.0"
+    "o-spacing": "^2.0.0",
+    "o-normalise": "^1.7.4"
```

Comment:
Need to mention this in MIGRATION.md

Reply:
Will do that soon.
>Future commits will improve the migration guide and update the readme.
As with all the components in the major cascade, we'll need a section that says
all of the dependencies have been bumped.